src/main/java/CinemaRoomRestService/RestService/domain/entity/Statistics.java

```diff
@@ -0,0 +1,66 @@
+package CinemaRoomRestService.RestService.domain.entity;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class Statistics {
```

Comment:
Let's get it clear:

Seat is your domain object it is your entity.
Statistics is not! Because you calculate it. There is no needs to save that object in database.
Use DTO object for containing information. 

Your Mapper returns StatisticsDTO, your mapper is able to call different services, and your mapper can use some helper services to Calculate values you need.

Do not forget About Builder Pattern to build your Object